crates/rpc/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -123,6 +123,7 @@ impl Transaction {
 
                 (Some(U128::from(gas_price)), Some(U128::from(signed_tx.max_fee_per_gas())))
             }
+            TxType::EIP4844 => todo!(),
```

Comment:
this can be moved to `TxType::EIP1559 | TxType::EIP4844` because use same dyn fees

Reply:
@mattsse Done